### DISCUSSION TITLE
HIKING TuYa DIN power meter new Ver.IVAP supported on EDGE but with errors

### DISCUSSION THREAD

**Discussion by gericho** [Koenkk/zigbee2mqtt#13822] [Devices]
I recentily bought 2 tuya DDS238-2, one two months ago, paired succesfully, and a new one, that appears to be identical, only with a more recent SN and Ver.IVAP written on the lower left corner, does not connect, Zigbee2MQTT says is unsupported. 
![DSC_1443](https://user-images.githubusercontent.com/29167042/188205894-2904011c-d02c-493d-8d1b-a11008a8ced7.JPG)

**Comment by gericho** [comment]
Will be possible to add compatibility to this version too? The vendor said they added more festure... What does it mean is a mystery. 
![Immagine 2022-09-05 105443](https://user-images.githubusercontent.com/29167042/188410025-63fff107-48a3-4bd6-ab41-0d9f3858ec67.png)

**Reply by darkkurt** [reply]
The same situation with the received device.

**Reply by Matriciel** [reply]
Idem for me

**Reply by gericho** [reply]
Any news? I update the EDGE every day with last converters but still no luck.

**Reply by Koenkk** [reply]
Can you provide the data/database.db entry for this device?
